reports/performance/_model.qmd

```diff
@@ -242,6 +242,235 @@ for (township in townships_to_iterate) {
 ```
 :::
 
+## How do Sales Compare to Estimates by Township?
+
+```{r _model_sales_vs_estimates_township, fig.width=8, fig.height=14}
+sales_vs_estimates <- model_performance_test %>%
+  filter(
+    triad_code == run_triad_code,
+    geography_type == "township_code",
+    by_class == FALSE
+  ) %>%
+  select(
+    geography_type,
+    geography_id,
+    num_sale,
+    sale_fmv_q25,
+    sale_fmv_median,
+    sale_fmv_q75,
+    estimate_fmv_q25,
+    estimate_fmv_median,
+    estimate_fmv_q75
+  )
+
+reshaped_data <- sales_vs_estimates %>%
+  mutate(township = ccao::town_convert(geography_id)) %>%
+  pivot_longer(
+    cols = -c(geography_type, geography_id, township, num_sale),
+    names_to = "type",
+    values_to = "value"
+  ) %>%
+  mutate(
+    category = ifelse(
+      grepl("sale", type),
+      paste(township, "- Sale"),
+      paste(township, "- Estimate")
+    )
+  ) %>%
+  select(category, value, num_sale) %>%
+  separate(category, into = c("township", "type"), sep = " - ") %>%
+  mutate(type_factor = factor(type, levels = c("Estimate", "Sale")))
+
+segment_data <- reshaped_data %>%
+  group_by(township, type) %>%
+  arrange(township, type, value) %>%
+  mutate(xend = lead(value)) %>%
+  filter(!is.na(xend))
+
+summary_data <- reshaped_data %>%
+  filter(type_factor == "Sale") %>%
+  group_by(township) %>%
+  summarize(
+    max_value = max(value),
+    num_sale = first(num_sale)
+  )
+
+ggplot() +
```

Comment:
**suggestions:** This is still a bit hard to read. For both the township and class versions, I would:

1. Drop the "Sale" and "Estimate" labels on the left axis, since they're redundant given the colored lines.
2. Color the points in addition to the lines.
3. Slightly increase the line width and color vibrancy.
4. Move the facet labels (township, class) to the left-hand side. Rotate the text counterclockwise 90 degrees.
5. Move the color legend to the bottom of the plot.
6. Right-align the counts and move them to the right-most possible position in the plot. Basically, align them with the right edge.

Reply:
Added all of these. Tried again for a while to put sale numbers outside of the right side of the graph but couldn't get there.